<a href="https://colab.research.google.com/github/Umizmar/The_theory_of_probability/blob/main/%D0%A3%D1%80%D0%BE%D0%BA_4_%D0%9D%D0%B5%D0%BF%D1%80%D0%B5%D1%80%D1%8B%D0%B2%D0%BD%D0%B0%D1%8F_%D1%81%D0%BB%D1%83%D1%87%D0%B0%D0%B9%D0%BD%D0%B0%D1%8F_%D0%B2%D0%B5%D0%BB%D0%B8%D1%87%D0%B8%D0%BD%D0%B0.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

##Урок 4. Непрерывная случайная величина

1. Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800]. Найдите ее среднее значение и дисперсию.


In [16]:
def mean_(a, b):
    return (a+b)/2

def disp(a, b):
    dispers = (b-a)**2/12
    return dispers

In [17]:
print('Среднее значение -', mean_(200,800))
print('Дисперсия -', disp(200,800))


Среднее значение - 500.0
Дисперсия - 30000.0


2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.


Т.к. $D(X)=\frac{(b-a)^2}{12}$, то  
$0.2=\frac{(b-0.5)^2}{12}\Rightarrow b-0.5=\sqrt2.4$

In [20]:
right = 2.4**0.5+0.5
print ('Правая граница', right)
print('Среднее значение', mean_(0.5, right))

Правая граница 2.049193338482967
Среднее значение 1.2745966692414834


3. Непрерывная случайная величина X распределена нормально и задана плотностью распределения $$f(x) = \frac{1}{4\sqrt2\pi}\cdot e^{-\frac{(x+2)^2}{32}}$$
Найдите:  
а). M(X)  
б). D(X)  
в). std(X) (среднее квадратичное отклонение)

Из общей формулы плотности нормального распределения:$$f(x) = \frac{1}{\sigma\sqrt2\pi}\cdot e^{-\frac{(x-a)^2}{2\sigma^2}}$$
Следует:  
$M(X)=a=-2$  
$std(X)=\sigma=4$  
$D(X)=\sigma^2=16$

4. Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.  
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:  
а). больше 182 см  
б). больше 190 см  
в). от 166 см до 190 см  
г). от 166 см до 182 см  
д). от 158 см до 190 см  
е). не выше 150 см или не ниже 190 см  
ё). не выше 150 см или не ниже 198 см  
ж). ниже 166 см.

Т.к. $z=\frac{x-m}{\sigma}=\frac{x-174}{8}$

In [1]:
def z(x):
    return (x-174)/8

а). Определим Z

In [2]:
print(z(182))

1.0


Смотрим значение по z-таблице: $0.84134$
Это значение z, которое меньше или равно 182см, т.к. нам надо больше, то

In [4]:
1-0.84134

0.15866000000000002

б).

In [3]:
print(z(190))

2.0


Смотрим значение по z-таблице: $0.97725$

In [5]:
1-0.97725

0.022750000000000048

в). z1 от 166  
z2 до 190


In [6]:
print(z(166))

-1.0


Смотрим значение по z-таблице:  0.1587

In [8]:
0.97725-0.1587

0.8185499999999999

г).

In [9]:
0.84134-0.1587

0.6826399999999999

д).

In [10]:
print(z(158))

-2.0


Смотрим значение по z-таблице: 0.0228

In [11]:
0.97725-0.0228

0.9544499999999999

е).

In [12]:
print(z(150))

-3.0


Смотрим значение по z-таблице: 0.0013

In [18]:
0.0013+0.022750000000000048

0.024050000000000047

ё).

In [19]:
print(z(198))

3.0


Смотрим значение по z-таблице: 0.99865

In [20]:
0.0013+1-0.99865

0.002650000000000041

а) $P(X\ge182)=0.1587$;  
б) $P(X\ge190)=0.02275$;  
в) $P(166\le X\le 190)=0.8186$;  
г) $P(166\le X\le 182)=0.6827$;  
д) $P(158\le X\le 190)=0.9545$;  
е) $P(X\le150\cup X\ge 190)=0.0241$;  
ё) $P(X\le150\cup X\ge 198)=0.0027$;  
ж) $P(X\le166)=0.1587$;  

5. На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

Т.к. $z=\frac{x-m}{\sigma}$  

$\frac{190-178}{\sqrt{25}}=\frac{12}{5}$

In [23]:
12/5

2.4